### DISCUSSION TITLE
Netbox v3 third Style looks V2.7

### DISCUSSION THREAD

**Discussion by skalec** [netbox-community/netbox#7378] [Ideas]
First of all, I would like to say great praise for this great software that you are developing. I found the visual style of netbox 2.7 great. Netbox V3 looks scary honestly, but that's just my opinion. Can you add a third theme based on V2.7?

**Comment by cpmills1975** [comment]
Personally I found many limitations in the UI for 2.x off the back of the Bootstrap 3 library and even contributed the odd fix or two to resolve them. For me BootStrap 5 is much better and I welcome the shiny new UI. It's not a theme, but a complete overhaul and I'd like to thank @jeremystretch and the other contributors for bringing this great piece of work forward.

**Comment by jeremystretch** [comment]
> Netbox V3 looks scary honestly

It would probably be more productive to suggest specific changes than simply stating that you don't like it. At any rate, we certainly won't be going back to BS3.

**Reply by tinyoverflow** [reply]
As I'm also not a fan of the new design, but I'll try to explain what specifically I don't like. The new design makes me feel unproductive.

**1. Scrolling Sidebar**
In my opinion a navigation bar (the sidebar in this case) should not be scrollable on a default Full HD screen. But due to the size of the sections and the spacing between individual items, it starts to scroll when opening Organization, Devices, IPAM and Other.
Example: IcingaWeb2 also uses a sidebar with quite some items, but it's way smaller and also it also has less spacing between some items:
<details>
  <summary>Example Screenshot</summary>

  ![grafik](https://user-images.githubusercontent.com/6287093/147754037-fe438d85-ce33-4029-b2d2-0af4c244d71b.png)

</details>


**2. Missing Contrast** 
In the new UI it is not quite clear where you are at the sidebar. I'm using the light UI as an example on this one. While the current section/navlink is highlighted in a light blue color, but it's pretty dense. With poor eyesight and maybe also a not so great monitor, this can be hard to realize at a first glance. Please take a look at the screenshot example in 1. to get an idea of what I mean with a better contrast.


**3. Color Hierachy in wrong order**
This is quite a tough one to give it a name, but in UI design the most important things (the content) should be easily visible while the less important things should be more in the background. Literally. In the current UI, the important parts (the content) is greyed out while the less important things (page title, header) are glowing in plain white. It should be the exact opposite. A great example is PatternFly by RedHat.
<details>
  <summary>Example Screenshot + PatternFly Screenshot</summary>

  ![grafik](https://user-images.githubusercontent.com/6287093/147754441-c3a46d47-d819-4c16-9a43-83cc690a54ac.png)

![grafik](https://user-images.githubusercontent.com/6287093/147754692-e6adea88-3f26-4f7a-9ce3-16a237b2f6e0.png)

</details>

**4. Hidden Search**
Hidden might not be the correct term, but in previous versions the search form was just at the right side of the entity list (devices, for example). Not it has its own page (tab) which requires another click to reach.


I hope this helps. Please feel free to get back to me anytime if you have questions or something's not clear enough.

**Reply by tinyoverflow** [reply]
About 10 months later, have you been able to get a look at the feedback, @jeremystretch?

**Reply by bluikko** [reply]
I have to say I agree on the point of overuse of whitespace. There was also a great issue about improving the top section of the actual content pages (whitespace here too, etc), too bad that one didn't get traction (either).

Edit: there is one major positive side on the v3 design which is: the horizontal screen real estate is used more efficiently for actual data tables. It could be improved but in short I think v3 was a success (could be improved more).
